packages/veraswap-app/src/routes/preview.tsx

```diff
@@ -337,9 +338,18 @@ function EmbedPreview() {
                                     {referrer && !isAddress(referrer) && (
                                         <p className="text-xs text-red-500">Please enter a valid address</p>
                                     )}
-                                    <p className="text-xs text-muted-foreground">
-                                        Enter a valid address to set as the referrer for this widget
-                                    </p>
+                                    <div className="space-y-2">
+                                        <p className="text-xs text-muted-foreground">
+                                            Enter a valid address to set as the referrer for this widget. The referrer{" "}
+                                            will receive a revenue share of all swap fees generated through this widget.
+                                        </p>
+                                        <div className="bg-muted p-2 rounded-md border">
+                                            <p className="text-sm font-semibold text-center">
+                                                Referral Fee:{" "}
+                                                {REFERRER_FEE_BIPS > 0n ? `${Number(REFERRER_FEE_BIPS) / 100}%` : "0%"}
```

Comment:
remove the ternary operator
If referrerer fee bips is 0n (default value), then 0/100 = 0 => formatting would still be 0% ?